curriculum/challenges/english/03-front-end-libraries/react/create-a-controlled-input.english.md

```diff
@@ -25,11 +25,11 @@ Last but not least, don't forget to add the necessary bindings in the constructo
 ```yml
 tests:
   - text: <code>ControlledInput</code> should return a <code>div</code> element which contains an <code>input</code> and a <code>p</code> tag.
-    testString: assert(Enzyme.mount(React.createElement(ControlledInput)).find('div').children().find('input').length === 1 && Enzyme.mount(React.createElement(ControlledInput)).find('div').children().find('p').length === 1, '<code>ControlledInput</code> should return a <code>div</code> element which contains an <code>input</code> and a <code>p</code> tag.');
+    testString: assert(Enzyme.mount(React.createElement(ControlledInput)).find('div').children().find('input').length === 1 && Enzyme.mount(React.createElement(ControlledInput)).find('div').children().find('p').length === 1);
   - text: The state of <code>ControlledInput</code> should initialize with an <code>input</code> property set to an empty string.
-    testString: assert.strictEqual(Enzyme.mount(React.createElement(ControlledInput)).state('input'), '', 'The state of <code>ControlledInput</code> should initialize with an <code>input</code> property set to an empty string.');
+    testString: assert.strictEqual(Enzyme.mount(React.createElement(ControlledInput)).state('input'), '');
```

Comment:
```suggestion
    testString: assert.strictEqual(Enzyme.mount(React.createElement(ControlledInput)).state('input'));
```

Reply:
@thecodingaviator We should not remove this one, because it is an `assert.strictEqual` which needs to compare to the empty string.

Reply:
Ohkay